eval.go

```diff
@@ -1700,6 +1721,7 @@ func (e *callExpr) eval(app *app, args []string) {
 		if !app.nav.init {
 			return
 		}
+		app.ui.sxScreen.updateSizes(app.ui.screen.Size())
```

Comment:
Is it possible to wrap this with `if gOpts.sixel { ... }` to avoid extra ioctl calls otherwise?